src/control/client/types.go

```diff
@@ -423,34 +422,64 @@ type StorageFormatResult struct {
 
 // AccessControlList is a structure for the access control list.
 type AccessControlList struct {
-	Entries []string // Access Control Entries in short string format
+	Entries    []string // Access Control Entries in short string format
+	Owner      string   // User that owns the resource
+	OwnerGroup string   // Group that owns the resource
 }
 
-// String converts the AccessControlList to a human-readable string.
-func (acl *AccessControlList) String() string {
-	var builder strings.Builder
-
-	builder.WriteString("# Entries:\n")
-	if acl.Empty() {
-		builder.WriteString("#   None\n")
-		return builder.String()
+// Empty checks whether there are any entries in the AccessControlList
+func (acl *AccessControlList) Empty() bool {
+	if acl == nil || len(acl.Entries) == 0 {
+		return true
 	}
+	return false
+}
 
-	for _, ace := range acl.Entries {
-		fmt.Fprintf(&builder, "%s\n", ace)
+// HasOwner checks whether the AccessControlList has an owner user.
+func (acl *AccessControlList) HasOwner() bool {
+	if acl == nil {
+		return false
 	}
 
-	return builder.String()
+	if acl.Owner != "" {
+		return true
+	}
+	return false
 }
 
-// Empty checks whether there are any entries in the AccessControlList
-func (acl *AccessControlList) Empty() bool {
-	if acl == nil || len(acl.Entries) == 0 {
+// HasOwnerGroup checks whether the AccessControlList has an owner group.
+func (acl *AccessControlList) HasOwnerGroup() bool {
+	if acl == nil {
+		return false
+	}
+
+	if acl.OwnerGroup != "" {
 		return true
 	}
 	return false
 }
 
+// String displays the AccessControlList in a basic string format.
+func (acl *AccessControlList) String() string {
+	if acl == nil {
+		return "nil"
```

Comment:
This was specifically added as debug output, which is used by the unit tests. I thought differentiating nil was useful in that case. I put the tool output formatting in dmg, rather than keeping it in the stringer as before. I wasn't planning to add a new stringer, but unit tests were trying to compare pointer addresses if I didn't implement one.